expr/payload.go

```diff
@@ -31,20 +33,64 @@ const (
 	PayloadBaseTransportHeader PayloadBase = unix.NFT_PAYLOAD_TRANSPORT_HEADER
 )
 
+// Possible PayloadCsumType values.
+const (
+	CsumTypeNone PayloadCsumType = unix.NFT_PAYLOAD_CSUM_NONE
+	CsumTypeInet PayloadCsumType = unix.NFT_PAYLOAD_CSUM_INET
+)
+
+// Possible PayloadType values.
+const (
```

Comment:
Related to (in netlink message):
[ payload write reg 1 => 4b @ network header + 16 ]
[ payload load 4b @ network header + 12 => reg 2 ]